tangelo/algorithms/classical/ccsd_solver.py

```diff
@@ -21,7 +21,7 @@
 from sympy.combinatorics.permutations import Permutation
 
 from tangelo.toolboxes.molecular_computation.molecule import SecondQuantizedMolecule
-from tangelo.toolboxes.molecular_computation import IntegralSolverPsi4, IntegralSolverPySCF
+from tangelo.toolboxes.molecular_computation import IntegralSolverPsi4, IntegralSolverPySCF, IntegralSolverPsi4QMMM
```

Comment:
Note: we do not seem to need `IntegralSolverPySCFQMMM` for some reason.

Reply:
We only need it for `IntegralSolverPsi4QMMM` because the frozen orbitals need to be defined before creating the molecule to use CCSD in Psi4. The other `IntegralSolverPsi4` and `IntegralSolverPySCF` are just used to check the `isinstance(molecule.solver)` to determine which CCSDSolver to use.